README.md

```diff
@@ -1 +1,15 @@
-# microverse_gitflow_start
\ No newline at end of file
+# My First Gitflow lesson
+
+This repository contains the starting lessons about Gitflow
+and Github actions first setup
+
+## Getting started
+
+Clone the repository using the command `git clone https://github.com/codedbystuart/microverse_gitflow_start.git`
+Navigate into the `microverse_gitflow_start` and run the index.html file
+
+You are good to go live 🔥
+
+## Contributors
+
+- Dambi Stuart
```

Comment:
- [ ] The README file should be descriptive all the main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in this repo 👌, so Kindly add the rest of the sections and customize it 💯
> We should always be sure our repositories are documented professionally, 💯